src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useCallback, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  todos: Todo[];
+  setTodos: (todos: Todo[]) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  todos,
+  setTodos,
+}) => {
+  const [todoTitle, setTodoTitle] = useState(todo.title);
+  const [editTitle, setEditTitle] = useState(false);
```

Comment:
Fix naming for all [boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)